src/main/java/graphql/schema/GraphqlDirectivesContainerTypeBuilder.java

```diff
@@ -31,17 +57,32 @@ public B withAppliedDirectives(GraphQLAppliedDirective... directives) {
         return (B) this;
     }
 
+    /**
+     * @param directive the directive to add
+     *
+     * @return this builder
+     *
+     * @deprecated - use {@link #addAppliedDirective(GraphQLAppliedDirective)} instead
+     */
+    @Deprecated(since = "2025-02-16", forRemoval = true)
     public B withAppliedDirective(GraphQLAppliedDirective directive) {
         assertNotNull(directive, () -> "directive can't be null");
         this.appliedDirectives.add(directive);
         return (B) this;
     }
 
+    /**
+     * @param builder the directive builder
+     *
+     * @return this builder
+     *
+     * @deprecated - use {@link #addAppliedDirective(GraphQLAppliedDirective.Builder)} instead
+     */
+    @Deprecated(since = "2025-02-16", forRemoval = true)
     public B withAppliedDirective(GraphQLAppliedDirective.Builder builder) {
         return withAppliedDirectives(builder.build());
```

Comment:
This may have been an unintentional typo during the major migration to Applied Directives.

The previous singular `withAppliedDirective` method that takes a `GraphQLAppliedDirective` does not clear any prior directives, but this `withAppliedDirective` with a builder parameter will take clear existing directives because it calls the plural `withAppliedDirectives` method